python/apertium_core.i

```diff
@@ -89,3 +131,13 @@ public:
   void postchunk_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
   void transfer_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
 };
+
+class tagger: public Apertium::apertium_tagger
+{
+public:
+  /**
+   * Imitates functionality of apertium-tagger
+   * tag::tag() passes int and char** to apertium_tagger::apertium_tagger() int&, char**& respectively
```

Comment:
I think the comments here and above are outdated. There really should be docs only at one spot.